tests/e2e/e2e_test.go

```diff
@@ -297,8 +297,42 @@ var _ = Describe("odoe2e", func() {
 				curProj = strings.TrimSuffix(curProj, "\n")
 				// Sleep until status tags and their annotations are created
 				time.Sleep(15 * time.Second)
+
+				if createFileAtPathWithContent(tmpDir+"/nodejs-ex/.odoignore", ".git\ntests/\nREADME.md") != nil {
+					fmt.Printf("the .odoignore file was not created, reason %v", err.Error())
+				}
+
 				runCmdShouldPass("odo create " + curProj + "/nodejs nodejs --local " + tmpDir + "/nodejs-ex")
 				runCmdShouldPass("odo push")
+
+				// get the name of the pod
+				podName := runCmdShouldPass("oc get pods | grep nodejs | awk '{print $1}' | tr -d '\n'")
+
+				// verify that the views folder got pushed
+				runCmdShouldPass("oc exec " + podName + " -- ls -lai /opt/app-root/src | grep views")
+
+				// verify that the tests was not pushed
+				runCmdShouldFail("oc exec " + podName + " -- ls -lai /opt/app-root/src | grep tests")
+
+				// verify that the README.md file was not pushed
+				runCmdShouldFail("oc exec " + podName + " -- ls -lai /opt/app-root/src | grep README.md")
```

Comment:
Then delete the freshly created ```.odoignore``` file. So that the next scenario ```It("should create a component and push using the --ignore flag", func() {}``` will get a clean environment to execute command ```runCmdShouldPass("odo push --ignore tests/,README.md")```. 

This is how we can confirm push with ```--ignore``` flag

Reply:
Yeah I noticed that. About to send a fix :)